hugr-core/src/hugr.rs

```diff
@@ -307,6 +308,25 @@ impl Hugr {
         // This step is not strictly necessary.
         self.graph.compact_nodes(|_, _| {});
     }
+
+    /// The underlying portgraph of the HUGR.
+    #[inline]
+    pub(crate) fn portgraph(&self) -> &MultiPortGraph {
+        &self.graph
+    }
+
+    /// Translate a node into an index in [`Hugr::portgraph`]
+    #[inline]
+    pub(crate) fn to_portgraph_node(&self, node: impl NodeHandle<Node>) -> portgraph::NodeIndex {
+        node.node().into_portgraph()
+    }
+
+    /// Translate a node index in [`Hugr::portgraph`] into a HUGR node.
+    #[allow(clippy::wrong_self_convention)]
+    #[inline]
+    pub(crate) fn from_portgraph_node(&self, index: portgraph::NodeIndex) -> Node {
+        index.into()
+    }
```

Comment:
I don't think these Node <-> NodeIndex conversions belong into `impl Hugr`. If I remember correctly, we added them here as a "hack" when the `Self::Node` AT was added to `HugrView`. Now might be a good time to remove them?